src/stabilization_report_template.md

```diff
@@ -0,0 +1,54 @@
+# Stabilization report template
+
+> **What is this?** This is a template to use for [stabilization reports](./stabilization_guide.md). It consists of a series of questions that aim to provide the information most commonly needed and to help reviewers be more likely to identify potential problems up front. Not all parts of the template will apply to all stabilizations. Feel free to put N/A if a question doesn't seem to apply to your case.
+
+## General design
+
+### What is the RFC for this feature and what changes have occurred to the user-facing design since the RFC was finalized?
+
+### What behavior are we committing to that has been controversial? Summarize the major arguments pro/con.
+
+### Are there extensions to this feature that remain unstable? How do we know that we are not accidentally committing to those?
+
+## Has a call-for-testing period been conducted? If so, what feedback was received?
+
+## Implementation quality
+
+### Summarize the major parts of the implementation and provide links into the code (or to PRs)
+
+An example for async closures: https://rustc-dev-guide.rust-lang.org/coroutine-closures.html
+
+### Summarize existing test coverage of this feature
+
+- What does the test coverage landscape for this feature look like?
+  - (Positive/negative) Behavioral tests?
+  - (Positive/negative) Interface tests? (e.g. compiler cli interface)
```

Comment:
the big categories I think are:
* tests for compiler errors when you use the feature wrongly or make simple mistakes
* tests for the feature itself:
  * tests for limits of the feature (so failing compilation)
  * run-pass tests that ensure that the feature works

Reply:
Used this wording instead